bazel/osx/crosstool/CROSSTOOL.tpl

```diff
@@ -1270,8 +1275,7 @@ toolchain {
     config_name: "c++-link-interface-dynamic-library"
     action_name: "c++-link-interface-dynamic-library"
     tool {
-      tool_path: "wrapped_clang"
-      execution_requirement: "requires-darwin"
+      tool_path: "DUMMY_TOOL"
```

Comment:
And I should add, I copied this CROSSTOOL.tpl from the 0.5.2 tag, hence the older DUMMY_TOOL entries.